packages/test-framework/src/fixture/pagePreset/AddItemToCart.ts

```diff
@@ -0,0 +1,19 @@
+import { Page } from '@playwright/test';
+
+import { ApiRequestsSender } from './ApiRequestsSender';
+import { CheckoutPagePreset } from './CheckoutPagePreset';
+
+export class AddItemToCart implements CheckoutPagePreset {
```

Comment:
This preset name may confuse devs because it still ends at `PaymentStepAsGuest`.
The additional call in `PaymentStepAsGuestPreset` is to ensure the wallet buttons will be presented.
Therefore, can we combine the two and use a toggle to control the addtional call?